model-api-gen-gradle-test/kotlin-generation/src/test/kotlin/org/modelix/modelql/typed/TypedModelQLTest.kt

```diff
@@ -360,4 +361,18 @@ abstract class TypedModelQLTest {
 
         assertTrue { actual.instanceOf(C_XmlDocument) }
     }
+
+    @Test
+    fun `buildQuery and callQuery can be separated`() = runTest { client ->
```

Comment:
```suggestion
    fun `buildQuery can be used outside of an ongoing query call`() = runTest { client ->
```

Maybe that's a bit more specific than "separate" (in time? in source code location?...)